torch/csrc/jit/passes/fold_conv_bn.cpp

```diff
@@ -353,12 +347,39 @@ void FoldConvBatchNorm2dHelper::transform() {
 
 } // namespace
 
-Module FoldConvBatchNorm2d(const Module& module) {
-  FoldConvBatchNorm2dHelper h;
+Module FoldConvBatchNorm(const Module& module) {
   Module m = module.clone();
-  addBiasForConv2dIfNone(m);
-  h.analyze(m);
-  h.transform();
+
+  addBiasForConvIfNone(m, "Conv2d");
+  addBiasForConvIfNone(m, "Conv3d");
+  // Conv2d + BatchNorm2d
+  const PatternInfo pattern2d = PatternInfo::parse_from_str(
+      R"(
+graph(%self, %x):
+    %conv_submodule = match::module[name=".Conv2d"](%self)
+    %conv_out = prim::CallMethod[name="forward"](%conv_submodule, %x)
+    %bn_submodule = match::module[name=".BatchNorm2d"](%self)
+    %bn_out = prim::CallMethod[name="forward"](%bn_submodule, %conv_out)
+    return (%bn_out))",
+      {is_conv2d_module, is_batchnorm2d_module});
```

Comment:
you'll need to update the pattern as well, `is_conv2d_module` is checking the type for `%conv`.
You can take a look at https://github.com/pytorch/pytorch/pull/39490/files to see how the patterns are updated

Reply:
the patterns should be something like:
```
graph(%x, %conv, %batchnorm):
    %conv_out = prim::CallMethod[name="forward"](%conv, %x)
    %bn_out = prim::CallMethod[name="forward"](%batchnorm, %conv_out)
    return (%bn_out))",
```

Reply:
Changed.